openedx/core/djangoapps/xblock/utils.py

```diff
@@ -67,3 +69,32 @@ def validate_secure_token_for_xblock_handler(user_id, block_key_str, token):
     # All computations happen above this line so this function always takes a
     # constant time to produce its answer (security best practice).
     return bool(result1 or result2)
+
+
+def get_id_for_anonymous_user(user):
+    """
+    Get a unique string that identifies the current anonymous (not logged in)
+    user. (This is different than the "anonymous user ID", which is an
```

Comment:
Nit: Is there any chance we can get another name for this?

Reply:
`xblock_id_for_anonymous_user`?

Reply:
Sure, change to `get_xblock_id_for_anonymous_user`